TwitterPolling/twitter-authorize.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------
+# twitter-authorize:
+#  - step through the process of creating and authorization a
+#    Twitter application.
+#-----------------------------------------------------------------------
+
+import twitter
+import time
+
+print("1. Create a new Twitter application at: https://apps.twitter.com\n")
+
+print("2. When you have created the application, enter:")
+print("   your application name:", end=' ')
+app_name = input()
+
+print("   your consumer key:", end=' ')
+consumer_key = input()
+
+print("   your consumer secret:", end=' ')
+consumer_secret = input()
+
+print()
+print("3. Now, we need to authorize this application.")
+print("   You'll be forwarded to a web browser in two seconds.\n")
+
+time.sleep(2)
```

Comment:
Why the sleep? I can't wait two seconds! So people know wtf is going on I guess?